blank-canvas-blocks/assets/ponyfill.css

```diff
@@ -267,14 +298,14 @@ ol {
 	padding: var(--wp--custom--navigation--submenu--padding);
 }
 
-@media only screen and (min-width: 482px) {
+@media (min-width: 480px) {
```

Comment:
Why was `only screen` removed? I'm assuming that's what core is doing, but I'm curious about the reason.

Reply:
I don't know the reason either, this is just following the way Gutenberg does it :)

Reply:
I think it would only matter if we wanted to do something specific for "screen" instead of "print" or perhaps for screen readers.  Probably gone not because it needs to be gone but because it doesn't need to be there. ¯\_(ツ)_/¯